extensions/http/src/test/java/org/mule/extension/http/BasicHttpTestCase.java

```diff
@@ -44,6 +44,8 @@
 import org.junit.Test;
 
 //TODO: MULE-9702 Remove once the tests are migrated.
+@ArtifactClassLoaderRunnerConfig(plugins = {"org.mule.modules:mule-module-sockets"},
+    providedInclusions = "org.mule.modules:mule-module-sockets")
```

Comment:
Because http needs sockets extension to be registered as plugin (it is already defined as provided dependency here) but also as "jar" in container, therefore it goes in both places. This would have to be migrated now that MULE-9867 added support for it and http should depend on sockets as a required extension plugin.